### PR TITLE
pythonPackages.fastparquet: 0.2.1 -> 0.3.2

### PR DESCRIPTION
**Pull request by veprbl**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

**Comment by veprbl** [comment]
@GrahamcOfBorg build python3Packages.fastparquet

**Comment by veprbl** [comment]
@GrahamcOfBorg build python3Packages.datashader

**Comment by veprbl** [comment]
@GrahamcOfBorg build python3Packages.fastparquet
@GrahamcOfBorg build python3Packages.datashader

File: pkgs/development/python-modules/fastparquet/default.nix [hunk 1]
```diff
@@ -27,8 +22,7 @@ buildPythonPackage rec {
   propagatedBuildInputs = [ numba numpy pandas thrift ];
   checkInputs = [ pytest python-snappy lz4 ];
 
-  # test_data/ missing in PyPI tarball
-  doCheck = false;
+  doCheck = true;
```
Comment (jonringer) [inline pkgs/development/python-modules/fastparquet/default.nix]: for buildPythonPackage, doCheck defaults to true
```suggestion
```

**Comment by veprbl** [comment]
@GrahamcOfBorg build python3Packages.fastparquet
@GrahamcOfBorg build python3Packages.datashader

**Pull request merged**